client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -36,13 +38,19 @@ import ScreenPatternList from './screen-pattern-list';
 import { encodePatternId } from './utils';
 import withGlobalStylesProvider from './with-global-styles-provider';
 import type { Pattern, Category } from './types';
-import type { Step } from '../../types';
+import type { StepProps } from '../../types';
 import type { OnboardSelect } from '@automattic/data-stores';
 import type { DesignRecipe, Design } from '@automattic/design-picker/src/types';
 import type { GlobalStylesObject } from '@automattic/global-styles';
 import './style.scss';
 
-const PatternAssembler: Step = ( { navigation, flow, stepName } ) => {
+const PatternAssembler = ( {
+	navigation,
+	flow,
+	stepName,
+	noticeList,
+	noticeOperations,
+}: StepProps & withNotices.Props ) => {
```

Comment:
`noticeList`,  `noticeOperations`, and `noticeUI` are props passed from the HOC `withNotices`. This PR doesn't use `noticeUI`. See [docs](https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/higher-order/with-notices).

We use `SnackbarList` to stack notices as in the editor. See [docs](https://wordpress.github.io/gutenberg/?path=/docs/components-snackbarlist--default).